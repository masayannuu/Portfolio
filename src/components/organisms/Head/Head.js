import React from 'react'
import { Image } from 'reactbulma'

import MainImage from '../../../../public/main-back.gif'
import Nav from '../../molecules/Nav/Nav'

const Head = () => (
  <div>
    <Nav />
    <Image src={MainImage} square="16by9" />
  </div>
)

export default Head
