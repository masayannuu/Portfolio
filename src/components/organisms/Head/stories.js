import React from 'react'
import { storiesOf } from '@storybook/react'

import Head from './Head'

storiesOf('Organisms.Head', module).add('Head', () => <Head />)
