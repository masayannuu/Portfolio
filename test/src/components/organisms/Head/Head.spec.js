import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Head from '../../../../../src/components/organisms/Head/Head'

describe('Head Head Molecules', () => {
  it('should render Head', () => {
    const wrapper = Enzyme.shallow(<Head />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
