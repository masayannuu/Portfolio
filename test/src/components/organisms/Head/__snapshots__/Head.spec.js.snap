// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head Head Molecules should render Head 1`] = `
<Head__StyledDiv>
  <Nav />
  <Head__StyledImage
    square="16by9"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</Head__StyledDiv>
`;
